src/Controller/Component/UsersAuthComponent.php

```diff
@@ -114,6 +118,38 @@ protected function _initAuth()
     }
 
     /**
+     * Check Public Access
+     *
+     * ### Usage
+     *
+     * Your permssions config should return '*' or 'public' for role(s) allowed.
```

Comment:
Don't worry about that, thinking about the issue and using direct language is something I like. 

I see your point using `public` and treating it as a regular role, so makes sense to have `*` allow public too if publicAcl is enabled.